### ISSUE TITLE
Root-level `security` declaration causes deserialization error

### ISSUE THREAD

**Issue by pilzm** [openapi4j/openapi4j#29]
**Describe the bug**
When using a root-level `security` block, the `specification.yaml` file can't be parsed correctly and fails with a 
```
Exception in thread "main" org.openapi4j.core.exception.ResolutionException: Failed to load spec at 'file:/.../minimal.yaml'
	at org.openapi4j.parser.OpenApi3Parser.parse(OpenApi3Parser.java:38)
	at org.openapi4j.parser.OpenApi3Parser.parse(OpenApi3Parser.java:19)
	at org.openapi4j.parser.OpenApiParser.parse(OpenApiParser.java:53)
	at org.openapi4j.parser.OpenApiParser.parse(OpenApiParser.java:37)
	at Reproducer.main(Reproducer.java:10)
Caused by: org.openapi4j.core.exception.DecodeException: Failed to decode : Cannot deserialize instance of `org.openapi4j.parser.model.v3.SecurityRequirement` out of START_ARRAY token
 at [Source: (StringReader); line: 15, column: 3] (through reference chain: org.openapi4j.parser.model.v3.OpenApi3["security"])
	at org.openapi4j.core.util.TreeUtil.load(TreeUtil.java:123)
	at org.openapi4j.parser.OpenApi3Parser.parse(OpenApi3Parser.java:35)
	... 4 more
```

I believe the issue can be resolved by converting the `security` field in `OpenApi3.java` to a `List<SecurityRequirement>`, the same way it's defined in `Operation.java`.

**To Reproduce**
1. Use a minimal specification with a root-level `security` block (see reproducer) and at least one security scheme
2. Load specification normally (see reproducer)
3. Observe exception

**Expected behavior**
I expect the root-level security block to behave the same as the operation-level ones and be parsed without problems.

**Additional context**
Version used: 0.5
Relevant OpenAPI doc: https://swagger.io/docs/specification/authentication/#security

**Reproducer**
[reproducer.zip](https://github.com/openapi4j/openapi4j/files/4040585/reproducer.zip)

**Comment by llfbandit** [comment]
Thank you for the report!

I fixed the typo in the model, it should be available on latest 0.6-SNAPSHOT.
Can you confirm the resolution with this publication?

**Comment by pilzm** [comment]
Thanks for the quick fix! I can confirm it works now :grinning:
